js/tests/unit/tooltip.spec.js

```diff
@@ -446,7 +446,7 @@ describe('Tooltip', () => {
       const tooltip = new Tooltip(tooltipEl)
       document.documentElement.ontouchstart = noop
 
-      spyOn(EventHandler, 'on')
+      spyOn(EventHandler, 'on').and.callThrough()
```

Comment:
I originally used EventHandler for the `executeAfterTransition` function and this was necessary to make the test work. 

Because using EventHandler in `util/index.js` caused some circular dependency warning (related to jQuery stuff), and a talk with @GeoSot, I changed the utility function to use `add/removeEventListener` instead.

It wouldn't harm to use it here but it shouldn't be necessary anymore, so I can change that back if desired.